src/main/java/com/foundation/view/AdvancedPanel.java

```diff
@@ -0,0 +1,271 @@
+/*
+ *  @(#)AdvancedPanel.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ *
+ */
+
+package com.foundation.view;
+
+import javax.swing.*;
+import javax.swing.border.Border;
+import java.awt.Dimension;
+import java.awt.GridBagLayout;
+import java.awt.GridBagConstraints;
+import java.awt.Insets;
+
+/*
+ * Panel class setting with grid definition
+ *
+ * @Author Ruben Mendoza
+ * @Version 1.0
+ */
+
+public class AdvancedPanel extends JPanel {
+    private JLabel extLabel, visibilityLabel, ownerLabel, fileSizeLabel, dateCreatedLabel, dateModifiedLabel, dateAccessedLabel;
```

Comment:
keep at the most 80 characters long in the same line

Reply:
This class will not be used at this point; its content will change and the fixes will be included when it will be implemented. But this issue is present in other Classes so I'll update them.